vm-builder/main.go

```diff
@@ -129,18 +139,28 @@ type inittabCommand struct {
 func main() {
 	flag.Parse()
 	var dstIm string
-
 	if *version {
 		fmt.Println(Version)
 		os.Exit(0)
 	}
-
 	if len(*daemonImageFlag) == 0 && len(NeonvmDaemonImage) == 0 {
 		log.Println("neonvm-daemon image not set, needs to be explicitly passed in, or compiled with -ldflags '-X main.NeonvmDaemonImage=...'")
 		flag.PrintDefaults()
 		os.Exit(1)
 	}
 
+	if targetArch == nil || *targetArch == "" {
+		log.Println("Target architecture not set, see usage info:")
+		flag.PrintDefaults()
+		os.Exit(1)
+	}
```

Comment:
IIUC, this will mean that we start requiring a new flag to be set — WDYT about alternately auto-detecting it here? (I'm personally undecided, want to hear what you think)